test/backup.ts

```diff
@@ -0,0 +1,550 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {PreciseDate} from '@google-cloud/precise-date';
+import * as promisify from '@google-cloud/promisify';
+import * as assert from 'assert';
+import {before, beforeEach, describe, it} from 'mocha';
+import * as proxyquire from 'proxyquire';
+import * as pumpify from 'pumpify';
+import {ServiceError} from 'google-gax';
+
+import * as clusterTypes from '../src/cluster';
+import * as backupTypes from '../src/backup';
+
+import {Bigtable} from '../src';
+
+let promisified = false;
+const fakePromisify = Object.assign({}, promisify, {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  promisifyAll(klass: Function, options: any) {
+    if (klass.name === 'Backup') {
+      promisified = true;
+      assert.deepStrictEqual(options.exclude, [
+        'endDate',
+        'expireDate',
+        'startDate',
+      ]);
+    }
+  },
+});
+
+describe.only('Bigtable/Backup', () => {
+  const BACKUP_ID = 'my-backup';
+  let CLUSTER: clusterTypes.Cluster;
+  let BACKUP_NAME: string;
+
+  let Backup: typeof backupTypes.Backup;
+  let backup: backupTypes.Backup;
```

Comment:
It seems if declare `backup` type `any` here, most subsequent casts to any (`(backup as any).someMethod = ...`) could be avoided?

Reply:
Nice, thanks! (Woops)